_docs-sources/library/overview/services.md

```diff
@@ -1,16 +1,36 @@
 # What is a service module?
 
-The Gruntwork Service Catalog consists of a number of customizable, production-grade infrastructure-as-code services that you can use to deploy and manage your infrastructure. This includes Docker orchestration, EC2 orchestration, load balancing, networking, databases, caches, monitoring, alerting, CI/CD, secrets management, VPN, and much more. Services combine multiple modules to configure an end-to-end solution.
+**Service modules are opinionated combinations of ["building block" modules](./modules). They are usually written in Terraform, represent a full use case, and are designed to be used "off the shelf."**
 
-## When should I use a service?
+## Example
 
-Using a service can save you time piecing together individual modules and testing that they’re correctly referencing each other. These are designed for specific use cases such as EKS and ECS clusters, VPCs with public and private subnets, and databases.
+Let’s look at an example service module. The [rds service module](/reference/services/data-storage/amazon-rds) creates not only an RDS database, but also a DNS record (so you can access the database with a human-friendly name), CloudWatch alarms (to alert you if something goes wrong), a CloudWatch dashboard (so you can see RDS KPIs at a glance), and a strategy for disaster recovery. The module does this by using many "building block" modules.
```

Comment:
```suggestion
Let’s look at an example service module. The [rds service module](/reference/services/data-storage/amazon-rds) creates an RDS database, a DNS record in Amazon Route53, CloudWatch alarms, a CloudWatch dashboard, and scheduled snapshots of the database. 

After applying, you will have an RDS instance that you can access via a human-friendly name, alarms to alert you if something goes wrong, a dashboard for viewing RDS KPIs, and a strategy for disaster recovery.
```

All of the parenthesis made this hard to read, so perhaps splitting this into two small paragraphs is more powerful?

Also, I don't think we need to redefine what a service module is, it's at the top of the page.

Reply:
Fixed in 54c7c0a705802bd656e58b914ad000e08b22640c!